matter_server/common/models.py

```diff
@@ -106,6 +109,74 @@ class ServerDiagnostics:
     events: list[dict]
 
 
+class NodeType(Enum):
+    """Enum with Matter node types."""
```

Comment:
This is somewhat confusing. While this are values stored in Matter enums, these note types are a pure Thread concept.

Reply:
I'd rather not as those are thread specific only. I want to make one value regardless if its wifi, thread or bridge

Reply:
as discussed this will be moved to client side only (just for HA)